go/pkg/global/variables.go

```diff
@@ -3,7 +3,7 @@ package global
 var Version = "development"     // set in Makefile
 var Environment = "development" // set in Makefile
 
-var ConfigFilename = "replicate.yaml"
+var ConfigFilename [2]string = [2]string{"replicate.yaml", "replicate.yml"}
```

Comment:
Can we rename this to `ConfigFilenames` now that we have more than one?

Reply:
Done